src/admin/components/ResourceTabs/AuditLogTab/components/SiteAuditLogEntityStatus.tsx

```diff
@@ -0,0 +1,79 @@
+import { FC } from "react";
+import { Link as RaLink, useBasename } from "react-admin";
+import { When } from "react-if";
+
+import modules from "@/admin/modules";
+import Text from "@/components/elements/Text/Text";
+import { AuditStatusResponse } from "@/generated/apiSchemas";
+
+import CommentarySection from "../../PolygonReviewTab/components/CommentarySection/CommentarySection";
+import { AuditLogButtonStates } from "../constants/enum";
+import { AuditLogEntity } from "../constants/types";
+import AuditLogTable from "./AuditLogTable";
+
+export interface SiteAuditLogEntityStatusProps {
+  entityType: AuditLogEntity;
+  record: SelectedItem | null;
+  auditLogData?: { data: AuditStatusResponse[] };
+  refresh: () => void;
+  buttonToogle: number;
+}
+
+interface SelectedItem {
+  title?: string | undefined;
+  name?: string | undefined;
+  uuid?: string | undefined;
+  value?: string | undefined;
+  meta?: string | undefined;
+  status?: string | undefined;
+}
+
+const SiteAuditLogEntityStatus: FC<SiteAuditLogEntityStatusProps> = ({
+  entityType,
+  record,
+  auditLogData,
+  refresh,
+  buttonToogle
+}) => {
+  const isSite = buttonToogle === AuditLogButtonStates.SITE;
+  const basename = useBasename();
+  const getTitle = () => {
+    if (!record?.title) {
+      return record?.name;
+    } else {
+      return record?.title;
+    }
+  };
```

Comment:
This can be simplified to `const getTitle = () => record?.title ?? record?.name;`